test-of-container-images-regarding-known-vulnerabilities.md

```diff
@@ -0,0 +1,16 @@
+# Test of Container Images Regarding Known Vulnerabilities
+Perform tests for your container images regarding known vulnerabilities.
+Note, that this can result in many found vulnerabilities. Check, whether they are fixed with the next build of the container image. See related Belt Activity: Container Update Enforcement.
```

Comment:
```suggestion
Note, that this can result in many found vulnerabilities. Check whether they are fixed with the next build of the container image. See related Belt Activity: Container Update Enforcement.
```